style.css

```diff
@@ -585,15 +587,31 @@ li[id^='can'] {
 
   .contact-ul input {
     width: 50%;
+    border-radius: 0;
+    border-style: none;
   }
```

Comment:
- As required by the previous reviewer to make sure your form looks like the Figma design, you've made some changes but there are still some issues with the width of the input as your's now is very big compared to the Figma design. Please adjust the width to match the design as much as you can below:

*your design*
![big](https://user-images.githubusercontent.com/59397159/179147072-851842d2-8242-47ee-8490-efbf3d5061bb.png)

*figma*
![figma](https://user-images.githubusercontent.com/59397159/179147115-89082bd1-7991-49a8-bd58-a3aa454089e3.png)

Reply:
resized the width of the form.

Reply:
<img width="1433" alt="Screenshot 2022-07-15 at 9 52 31 AM" src="https://user-images.githubusercontent.com/85638137/179149943-2cc50327-53cf-48c0-bd40-3eb1577b3b40.png">